### ISSUE TITLE
Extra information on training

### ISSUE THREAD

**Issue by thijsgelton** [IPMI-ICNS-UKE/multiscale#2]
Hi!

Thank you for your awesome contributions. I was wondering whether I overlooked any of the training parameters. Could you provide a list of what hyperparameters and such, so I can try to recreate the scores you've mentioned in the paper?

**Comment by RSchmitzHH** [comment]
Yes, of course! Would you send me a message via r dot schmitz at uke dot de, so we can see what parameters you are missing? I've seen that you want to integrate our model into some kind of a self-configuring nnUnet-like pipeline for histopathology? Sound cool!

**Comment by thijsgelton** [comment]
Hi,

My apologies for the late reply. I have been working on different projects and just now returned to working on this one. I will send you an e-mail with other questions as well, but I think it is informative for others to know how exactly you obtained the reference standard for the context classification loss. In your paper you state "_The classification loss is computed with respect to the content of the detail patch_". How exactly is this then computed?

Thank you.

**Comment by thijsgelton** [comment]
Would you be able to supply the CV split for the PAIP2019 dataset?

**Comment by thijsgelton** [comment]
Hi again 😅,

What was your reasoning behind pre-sampling four patches of size 3072 per WSI? Why not more patches with smaller patch size?

**Comment by RSchmitzHH** [comment]
> Hi,
> 
> My apologies for the late reply. I have been working on different projects and just now returned to work on this one. I will send you an e-mail with other questions as well, but I think it is informative for others to know how exactly you obtained the reference standard for the context classification loss. In your paper you state "_The classification loss is computed with respect to the content of the detail patch_". How exactly is this then computed?
> 
> Thank you.

We computed the overall contents of the detail patch, i.e., let the encoder predict all classes that are present anywhere in the detail patch. The loss is then computed as BCE (with logits). 

In the respective models (with "class" in their name), the logits (or probas, depending on the pred_act) for the contents of the detail patch are returned as x_tail.

**Comment by RSchmitzHH** [comment]
> Hi again 😅,
> 
> What was your reasoning behind pre-sampling four patches of size 3072 per WSI during validation? Why not more patches with smaller patch size?

We experimentally found that this gave a good estimate of the overall performance. More patches of smaller size would of course also be possible. Just keep in mind that border effects may play a more important role (which may differ between models).

**Comment by RSchmitzHH** [comment]
> Would you be able to supply the CV split for the PAIP2019 dataset?

See e-mail :)
